ebill_postfinance_stock/tests/examples/invoice_qr_yb.xml

```diff
@@ -0,0 +1,192 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
FYI if you want to avoid having linters to run here you should rename the folder to `samples`

Reply:
Good to know, thanks.